<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="input">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="1">
        <widget class="QLabel" name="publicLabel">
         <property name="text">
          <string>Public key:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="eLabel">
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="eLineEdit">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="nLabel">
         <property name="text">
          <string>n</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="nLineEdit">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="dLabel">
         <property name="text">
          <string>d</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="dLineEdit">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="nLabel_2">
         <property name="text">
          <string>n</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="nLineEdit_2">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="privateLabel">
         <property name="text">
          <string>Private key:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate keys</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignRight">
        <widget class="QRadioButton" name="encodeRadioButton">
         <property name="text">
          <string>Encode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertButton">
         <property name="text">
          <string>Convert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="decodeRadioButton">
         <property name="text">
          <string>Decode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
